admin/includes/stylesheet.css

```diff
@@ -8,6 +8,10 @@
 
 @import url("css/bootstrap.min.css");
 /* @import url("https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"); */
+.form-control {

```

Comment:
Why do we need this?

Reply:
Otherwise the * would appear on the next line.  You can try it yourself by modifying a simple form like banner manager to add the required flag.

Reply:
ok. that's valid, and easier than adding a new div or span. 👍

Reply:
Thanks @Zen4All  and BTW, thanks for the huge improvement to how admin pages look in 1.5.6.  Really makes the product look more up to date.

Reply:
Would up having to move this stuff to the bottom - it was messing with the menu.